docs/sources/alerting/set-up/configure-roles/index.md

```diff
@@ -64,3 +64,26 @@ To manage folder permissions, complete the following steps.
 1. Hover your mouse cursor over a folder and click **Go to folder**.
 1. Click **Manage permissions** from the Folder actions menu.
 1. Update or add permissions as required.
+
+## Manage access using contact point permissions
+
+Available only when feature flag `alertingApiServer` is enabled.
+
+You can extend or limit the access provided by a role to contact points by assigning permissions to individual contact point.
+
+This allows different users, teams, or service accounts to have customized access to read or modify specific contact points.
+
+Refer to the following table for details on the additional access provided by contact point permissions:
+
+| Folder permission | Additional Access                                                                                                                            |
+| ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| View              | Ability to view and export contact point as well as select it in Alert rule edit page                                                        |
+| Edit              | Update or delete the contact point                                                                                                           |
+| Admin             | Same additional access as Edit and manage permissions for the contact point. User should have additional permissions to read users and teams |
+
+To contact point permissions, complete the following steps.
+
```

Comment:
this is only for admins right? or can other users change permissions? isn't there some rule that you can edit if you created the contact point, too?

Reply:
This is for all users who can administer a contact point. Basic "Admin" role can do that for all. User that creates a new contact point is automatically granted admin permissions to it, and therefore that user can grant admin permissions to other users\teams